### PR TITLE
Strip attemptId from bucket file for delete delta

### PR DESCRIPTION
**Pull request by losipiuk**

### CODE REVIEW

**Review by djsstarburst — APPROVED** [review]
Looks good.  Thanks for the quick work!

File: presto-hive/src/main/java/io/prestosql/plugin/hive/HiveUpdatablePageSource.java [hunk 1]
```diff
@@ -66,7 +67,7 @@
     public static final Block DELETE_OPERATION_BLOCK = nativeValueToBlock(INTEGER, Long.valueOf(DELETE.getOperationNumber()));
 
     // The bucketPath looks like .../delta_nnnnnnn_mmmmmmm_ssss/bucket_bbbbb(_aaaa)?
-    public static final Pattern BUCKET_PATH_MATCHER = Pattern.compile("(?s)(?<rootDir>.*)/(?<dirStart>delta_[\\d]+_[\\d]+)_(?<statementId>[\\d]+)/(?<filename>bucket_(?<bucketNumber>[\\d]+)(?<attemptId>_[\\d]+)?)$");
+    public static final Pattern BUCKET_PATH_MATCHER = Pattern.compile("(?s)(?<rootDir>.*)/(?<dirStart>delta_[\\d]+_[\\d]+)_(?<statementId>[\\d]+)/(?<filenameNoAttempt>bucket_(?<bucketNumber>[\\d]+))(?<attemptId>_[\\d]+)?$");
```
Comment (findepi) [inline presto-hive/src/main/java/io/prestosql/plugin/hive/HiveUpdatablePageSource.java:70]: filenameNoAttempt -> filenameBase ?

**Review by findepi — APPROVED** [review]
we could have a unit test for this, perhaps somewhat similar to TestOrcDeleteDeltaPageSource

File: presto-hive/src/main/java/io/prestosql/plugin/hive/HiveUpdatablePageSource.java (hunk 1 above)
Comment (losipiuk) [reply presto-hive/src/main/java/io/prestosql/plugin/hive/HiveUpdatablePageSource.java:70]: I will change after CI pass

**Comment by losipiuk** [comment]
> we could have a unit test for this, perhaps somewhat similar to TestOrcDeleteDeltaPageSource

We could. Let's make it a followup. I do not have enough time to cover it today and it would be nice to have this PR in 345 release.

**Review by electrum — APPROVED** [review]

File: presto-hive/src/main/java/io/prestosql/plugin/hive/HiveUpdatablePageSource.java [hunk 2]
```diff
@@ -28,6 +28,7 @@
 import io.prestosql.spi.type.TypeManager;
 import org.apache.hadoop.conf.Configuration;
 import org.apache.hadoop.fs.Path;
+import org.jetbrains.annotations.NotNull;
```
Comment (electrum) [inline presto-hive/src/main/java/io/prestosql/plugin/hive/HiveUpdatablePageSource.java:31]: This seems to be unused

Comment (losipiuk) [reply presto-hive/src/main/java/io/prestosql/plugin/hive/HiveUpdatablePageSource.java:31]: Yeah - I have a fixup for this. Just waiting for CI.

**Pull request merged**